// Copyright (c) 2025 Advanced Micro Devices, Inc. All Rights Reserved.
syntax = "proto3";

package Schola;

enum DType 
{
    FLOAT32 = 0;
    UINT8 = 1;
    UINT16 = 2;
    UINT32 = 3;
    UINT64 = 4;
    INT8 = 5;
    INT16 = 6;
    INT32 = 7;
    INT64 = 8;
    FLOAT16 = 9;
    FLOAT64 = 10;
    BOOL = 11;
}